templates/call-for-papers.html

```diff
@@ -6,13 +6,65 @@
 {% endblock %}
 
 {% block tabs %}
-{% endblock %}
-
-{% block content %}
 <div class="page-header text-center">
   <h1>Call for Papers</h1>
 </div>
+{{ components.tabs([("author-info", "Author Information", "active"),
+                    ("track-1", "Track 1: Machine Learning", ""),
+                    ("track-2", "Track 2: Applications and Practice", ""),
+                    ("track-3", "Track 3: Impact and Society", ""),
+                    ("chil-2020-summary", "CHIL 2020 Summary", ""),
```

Comment:
"CFP 2020" might be better (although I realise this is temporary)

Reply:
Removed in favour of highlights page.